CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - The server no longer crashes, when received RPCs are processed recursively.
 - DeploymentLauncher can parse a .pb.json launch configuration.
 
+### Features:
+- Added two scripts to the UnrealExampleProject: LaunchSimPlayerClient.bat and LaunchSimPlayers.bat. LaunchSimPlayerClient.bat will launch a single SimPlayer client. LaunchSimPlayers.bat will launch 10 SimPlayer clients.
```

Comment:
```suggestion
- In local deployments of the Example Project you can now launch Simulated Players in one click. Running `LaunchSimPlayerClient.bat` will launch a single Simulated Player client. Running `LaunchSimPlayers.bat` will launch 10.
```

Reply:
This better conforms to the WIIFM format advocated in: [how-to-write-good-release-notes.md](https://github.com/spatialos/UnrealGDK/blob/release/SpatialGDK/Extras/internal-documentation/how-to-write-good-release-notes.md).

Reply:
Ooooh, nice.